include/seqan3/io/structure_file/input.hpp

```diff
@@ -66,11 +66,11 @@ namespace seqan3
  * \{
  */
 /*!\typedef using seq_alphabet
- * \memberof seqan3::structure_file_input_traits_concept
+ * \memberof seqan3::StructureFileInputTraits
```

Comment:
If the group is already \memberof shouldn't we remove the individual \memberof-lines as per the requirements of the new doxygen?

Reply:
Yes!

Reply:
Oh but I think there is already a PR for file io so those two might clash

Reply:
So, I wait and rebase or the other PR rebases?

Reply:
If you are faster we will merge this first :D
(remove the extra memberof's)